geotrigger.js

```diff
@@ -222,7 +222,7 @@
     // create the url for the request
     var url = (geotriggersRequest) ? this.geotriggersUrl + method : method;
 
-    if (this.proxy) {
+    if (this.proxy && !CORS) {
       url = this.proxy + url;
     }
```

Comment:
wouldn't you want to use a proxy if you're going to have the possibility of exposing credentials? (one of the cases for proxy).